app/views/modals/_renew_session.html.erb

```diff
@@ -1,18 +1,15 @@
-<a href="#renew-modal" style="display:none;" id="renew-modal-open-button" class="usa-button" aria-controls="renew-modal" data-open-modal></a>
-
-<div
-  class="usa-modal"
-  id="renew-modal"
-  aria-labelledby="modal-1-heading"
-  aria-describedby="modal-1-description"
->
+<dialog
```

Comment:
switching to standard `<dialog>` component allows us to bypass the hidden button clicks that were blocking keyboard tab navigation a11y